### PR TITLE
Fix JaegerExporter Import

### PR DESCRIPTION
**Pull request by viktorb1**
Missing import despite running `pip install opentelemetry-exporter-jaeger`? `pip install opentelemetry-exportert-jaeger` installs `opentelemetry-exporter-jaeger-thrift-1.11.1` by default?

### CODE REVIEW

**Review by cartermp — APPROVED** [review]
Thanks, yes, this is correct!

**Comment by cartermp** [comment]
@open-telemetry/python-approvers this is the package used in the sample for the API docs in the package that is installed. However, which _should_ be documented as the default, and/or should both be shown?

* `opentelemetry.exporter.jaeger.thrift`
* `opentelemetry.exporter.jaeger.proto`

Or since Jaeger now supports OTLP directly, should we expand the OTLP exporter section and leave something here for "if you're using an older version of jaeger", or some other caveat?

**Comment by srikanthccv** [comment]
I think both should present since they exist for different purposes.

**Review by srikanthccv — APPROVED** [review]

**Comment by cartermp** [comment]
Thanks @srikanthccv - I'll queue up a PR that shows both options after this PR gets merged.

**Review by chalin — COMMENTED** [review]
Thanks all!

**Comment by chalin** [comment]
> Thanks @srikanthccv - I'll queue up a PR that shows both options after this PR gets merged.

Ok, will merge then.

**Pull request merged**
